Games/types/Mafia/roles/cards/WinWithMafia.js

```diff
@@ -50,6 +51,25 @@ module.exports = class WinWithMafia extends Card {
           );
         }
 
+        // win by killing president
+        var hasPresident = false;
```

Comment:
if there's 2 presidents does mafia win when killing one or both

Reply:
must kill only one to win